POC/api/main.py

```diff
@@ -160,23 +423,25 @@ async def create_new_database_field(database_id: int):
     response_model_exclude_none=True,
     include_in_schema=False,
 )
-async def create_database_field(
-    form: Annotated[FieldInfoForm, fastui_form(FieldInfoForm)], database_id: int
+async def submit_field_form(
+    field_form: Annotated[FieldInfoForm, fastui_form(FieldInfoForm)], database_id: int
 ):
     with Session(db_engine) as session:
-        db = FieldInfo(**form.model_dump(), db_id=database_id)
-        session.add(db)
+        db_field = FieldInfo(**field_form.model_dump(), db_id=database_id)
+        session.add(db_field)
         session.commit()
-        session.refresh(db)
+        session.refresh(db_field)
+    with Session(db_engine) as session:
+        field_statement = select(FieldInfo).where(FieldInfo.db_id == database_id)
+        db_fields = session.exec(field_statement).all()
+        display_db_fields = [FieldInfoForm(**field.model_dump()) for field in db_fields]
```

Comment:
is there a faster or better way to convert a list of FieldInfo to a list of FieldInfoForm or should this list comprehension work?